### PR TITLE
Buildkite plugin

### PR DESCRIPTION
**Pull request by markkemper1**
Hi, 

I'm not expecting you to merge this 😄 

I've added a plugin for buildkite & a new 'minimal' theme.

Looking for feedback & if your interested in including this plugin or theme. 

Thanks,

### CODE REVIEW

**Comment by marcells** [comment]
Hey, thank you. :-) I'll review it tonight or tomorrow.

File: app/config.json [hunk 1]
```diff
@@ -1,22 +1,15 @@
 {
```
Comment (marcells) [inline app/config.json]: Please leave this file as it is. It is used to show some default builds with the default theme.

File: app/services/Buildkite.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              RUNNING: { desc: " The build is currently running jobs", color: "#ffa500" },
+              PASSED: { desc: "The build passed", color: "#008000" },
+              FAILED: { desc: "The build failed", color: "#ff0000" },
+              CANCELING: { desc: "The build is currently being canceled", color: "#ffb3b3" },
+              CANCELED: { desc: "The build was canceled", color: "#ff4d4d" },
+              BLOCKED: { desc: "The build is blocked", color: "#003300" },
+              NOT_RUN: { desc: "The build wasn't run", color: "#808080" }
+            };
+
+            return {
```
Comment (marcells) [inline app/services/Buildkite.js]: Please make it compatible to the default theme.

**Comment by markkemper1** [comment]
Created a seperate pull request, as requested.

**Pull request closed without merging**